### PR TITLE
fix(readme): standardize format + link to docs

### PR DESCRIPTION
**Pull request by ami**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,671 +56,86 @@ lob.api_key = 'your-api-key'
...
-    "address_line2": "",
-    "address_state": "CA",
-    "address_zip": "94107-5705"
-  },
-  "message": "Default address: The address you entered was found but more information is needed (such as an apartment, suite, or box number) to match to a specific address."
-}
-```
+### Usage
+
+We've provided an example script you can run in examples/ that has examples ohow to use the lob-python wrapper with some of our core endpoints.
```
Comment (elnaz) [inline README.md]: ...examples of how to...

**Comment by ami** [comment]
@elnaz updated!

**Comment by elnaz** [comment]
LGTM. I have to merge the bank verify endpoint PR before this so I'll take care of merging this too.

**Comment by leore** [comment]
Other PR is in. I think if we rebase this will now pass.

**Comment by leore** [comment]
LGTM (merge away) @pon can do you want to show how to publish

**Pull request merged**